os/include/tinyara/fs/ioctl.h

```diff
@@ -94,6 +94,7 @@
 #define _FOTABASE       (0x1900)	/* FOTA ioctl commands */
 #define _GPIOBASE       (0x2000)	/* GPIO ioctl commands */
 #define _TMBASE         (0x2100)	/* Task Management ioctl commands */
+#define _ERBASE			(0x2200)	/* Error Report ioctl commands */
```

Comment:
Please align

Reply:
Aligned using spaces instead of tab. Thanks.